packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -1248,6 +1264,14 @@ export const DEFAULT_GAS_SCHEDULE: Required<FeeSchedule> = {
         }
         return gas;
     },
+    [ERC20BridgeSource.UniswapV3]: (fillData?: FillData) => {
+        let gas = 260e3;
```

Comment:
:money_with_wings:

Reply:
The 260 includes the FW overhead? 

```
SELL WETH->YEENUS 0.01 -> 0.01 ($0.01) after 5.0s @ http://localhost:7001/swap/dev/quote?includedSources=Uniswap_V3
	✔ PASS 0.01 -> 0.01 ($0.01) ($0.00)
	Uniswap_V3: 100%
	gas: 258899
```

This should be 260-130 or something right? 260 sounds high.

In the future we should calculate cumulative `ticks` crossed and return this in `fillData`, replacing path length with this should be more accurate.

Reply:
yeah I shaved off 100k now

Counting ticks is pretty involved, which is why the quoter just does a swap and revert. But because it's just a swap and revert we can probably use the gas used as a baseline of actual gas usage.